packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -36,7 +37,7 @@ function toMatchReference(screenshot) {
   }
 
   return {
-    pass: imagesMatch === true,
+    pass: disableSnapshotMismatch ? true : imagesMatch === true,
```

Comment:
my vote is to use the logical OR operator to make this less verbose
```suggestion
    pass: disableSnapshotMismatch || imagesMatch,
```

Reply:
Changed here https://github.com/cerner/terra-toolkit/pull/655/commits/8571fc4e7f0de6b5aa3178eb5c70084c3fa80539#diff-1372ed3653dff72eae9a8aa8b3969b4e50473f645766c7c2f232298e514d0518R39